app/src/main/java/org/oppia/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -88,12 +96,12 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
     if (previousMenuItemId != menuItemId && menuItemId != 0) {
       when (NavigationDrawerItem.valueFromNavId(menuItemId)) {
         NavigationDrawerItem.HOME -> {
-          val intent = Intent(fragment.activity, HomeActivity::class.java)
+          val intent = HomeActivity.createHomeActivity(activity, internalProfileId)
           fragment.activity!!.startActivity(intent)
           fragment.activity!!.finish()
         }
         NavigationDrawerItem.HELP -> {
-          val intent = Intent(fragment.activity, HelpActivity::class.java)
+          val intent = HelpActivity.createHelpActivityIntent(activity, internalProfileId)
```

Comment:
In this you don't need to pass `interalProfileId`

Reply:
@rt4914 I have checked the implementation, the `NavDrawerFragmentPresenter `receives an Integer profileId which is then converted to `ProfileId `class in the presenter. Hence, I have passed the profileId in integer form which is `internalProfileId `in above code.